README.md

```diff
@@ -46,3 +34,7 @@ Unofficial native OpenRepos.net client for Sailfish OS
 ## Translations
 
 Read [here](harbour-storeman/translations/README.md)
+
+## Installing Storeman
+
+A changelog, screenshots and installable RPMs are available at [OpenRepos.net](https://github.com/mentaljam/harbour-storeman/edit/master/README.md).
```

Comment:
The full changelog is available only here on GitHub. Also you have messed up the link.